angr/analyses/reaching_definitions/definition.py

```diff
@@ -10,9 +10,11 @@ class Definition:
     An atom definition.
 
     :ivar Atom atom:            The atom being defined.
-    :ivar CodeLocation codeloc: Where this definition is created.
-    :ivar data:                 A concrete value (or many concrete values) that the atom holds when the definition is
+    :ivar CodeLocation codeloc: Where this definition is created in the original binary code.
```

Comment:
Am not sure if I should specify `angr.analyses.code_location.CodeLocation` instead of simply `CodeLocation`.

I have gone for the full namespaced name in some places (see under, for example in `LiveDefinitions`). So as is, it's rather inconsistent.

Thoughts?

Reply:
Poke @rhelmot @ltfish .

Reply:
You have to specify the full namespaced value or else sphinx won't generate the link for it. It's ok to not have links sometimes but idk where the line is